src/style.css

```diff
@@ -1 +1,74 @@
 /* Styles go here */
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+  font-family: "Roboto", "Montserrat", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  background-color: white;
```

Comment:
And delete this background-color. If there isn't another value, white is default.